### PR TITLE
[8.15] Endpoint API documentation for `scan` response action

### PR DESCRIPTION
**Pull request by ashokaditya**
Adds API documentation for `scan` response action.

### Preview of updated/added pages
- https://security-docs_bk_5476.docs-preview.app.elstc.co/guide/en/security/master/scan-api.html

### CODE REVIEW

File: docs/management/api/scan-api.asciidoc [hunk 1]
```diff
@@ -0,0 +1,89 @@
+[[scan-api]]
+=== Scan a file or folder
+
+Scan a file or folder on a host running {elastic-defend} for malware.
+
+You must have the `Scan Operations` {kib} privilege in the Security feature as part of your role and at least an Enterprise license to perform this action.
```
Comment (paul-tavares) [inline docs/management/api/scan-api.asciidoc:6]: Maybe remove "...at least..." since `Enterprise` is the highest license a company can have.

**Review by paul-tavares — APPROVED** [review]

File: docs/management/api/scan-api.asciidoc (hunk 1 above)
Comment (ashokaditya) [reply docs/management/api/scan-api.asciidoc]: Done d48f7883bfcd5facae8c638c2da4ad62de495e1b

**Review by paul-tavares — APPROVED** [review]

File: docs/management/api/scan-api.asciidoc [hunk 2]
```diff
@@ -0,0 +1,89 @@
+[[scan-api]]
+=== Scan a file or folder
+
+Scan a file or folder on a host running {elastic-defend} for malware.
```
Comment (joepeeples) [inline docs/management/api/scan-api.asciidoc:4]: The end bit "for malware" sounds like the host must be running Defend AND have "Malware protections" enabled, but I'm not sure that's the case? Maybe it means we're scanning for malware:
```suggestion
Scan a file or folder for malware on a host running {elastic-defend}.
```

File: docs/management/api/scan-api.asciidoc [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+|==============================================
+include::_response-actions-api-reusable-content.asciidoc[tags=create-response-actions-api-common-body-options-row-content]
+
+|`parameters.path` |String |The folder or file’s full path (including the file name). |Yes
+|==============================================
+
+
+===== Example requests
+
+Scan a file `/usr/my-file.txt` on a host with an `endpoint_id` value of `ed518850-681a-4d60-bb98-e22640cae2a8` and comments `Scan the file for malware`:
```
Comment (joepeeples) [inline docs/management/api/scan-api.asciidoc:26]:
```suggestion
Scan a file `/usr/my-file.txt` on a host with an `endpoint_id` value of `ed518850-681a-4d60-bb98-e22640cae2a8` and comment `Scan the file for malware`:
```

File: docs/management/api/scan-api.asciidoc [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+--------------------------------------------------
+
+
+===== Response code
+
+`200`::
+   Indicates a successful call.
+
+`403`::
+	Indicates insufficient privileges, or unsupported license level (minimum Enterprise license required).
```
Comment (joepeeples) [inline docs/management/api/scan-api.asciidoc:47]:
```suggestion
	Indicates insufficient privileges, or unsupported license level (Enterprise license required).
```

**Review by joepeeples — COMMENTED** [review]
Added a few small edit suggestions. Thanks!

**Review by paul-tavares — APPROVED** [review]
👍

**Review by joepeeples — APPROVED** [review]

**Pull request merged**